artifacts/definitions/Server/Enrichment/JsonLookup.yaml

```diff
@@ -0,0 +1,36 @@
+name: Server.Enrichment.JsonLookup
+description: Allows pulling in JSON lists from an external URL to perform lookups against
+author: Whitney Champion (@shortxstack)
+type: SERVER
+
+parameters:
+    - name: LookupUrl
+      type: string
+      default: 
+
+sources:
+  - query: |
+  
+        // Assumes JSON is formatted like this:
+        // {
+        //     "list":
+        //     [
+        //         {"key":"value"},
+        //         {"key":"value"}
+        //     ]
+        // }
+
+        LET Data = SELECT parse_json(data=Content) AS Lookup
+            FROM http_client(url=LookupUrl, headers=dict(`Accept`="application/json"), method='GET')
+
+        SELECT 
+            Lookup.list AS Lookup
+        FROM Data
+        
+        // 
+        // Notebook usage:
```

Comment:
@scudette can we do notebook suggestions with server artifacts?
It would be cool to drop down notebook suggestion for this and other artifacts we want to roll reguarly.

Reply:
Yeah we should be able to do notebook suggestions with all these. Maybe this better go in the exchange or in the knowledge base?